packages/coin-support-utils/tests/02.createAccount/03.createAccountNameGenerator/index.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, test } from '@jest/globals';
+import { createAccountNameGenerator } from '../../../src';
+import { fixtures } from './__fixtures__';
+
+describe('createAccountNameGenerator', () => {
```

Comment:
Keep the description consistent. You can use the format `<Serial Number>. <Function Name>` or `<Function Name>` for all tests